SwiftProtobuf.podspec

```diff
@@ -8,10 +8,11 @@ Pod::Spec.new do |s|
   s.source = { :git => 'https://github.com/gojekfarm/swift-protobuf.git', :tag => s.version }
 
   s.requires_arc = true
-  s.ios.deployment_target = '9.0'
-  s.osx.deployment_target = '10.9'
-  s.tvos.deployment_target = '9.0'
-  s.watchos.deployment_target = '2.0'
+  s.ios.deployment_target = '11.0'
+  s.osx.deployment_target = '10.13'
+  s.tvos.deployment_target = '11.0'
+  s.watchos.deployment_target = '4.0'
+  s.visionos.deployment_target = "1.0"
 
   s.cocoapods_version = '>= 1.7.0'
```

Comment:
Let's change this to:

`'>= 1.13.0'`

Reply:
Asking for this because [this commit](https://github.com/gojekfarm/swift-protobuf/pull/3/commits/a04f7766cb07151c31cdf6429a6ec0d2c4672c16) had downgraded this version.